pom.xml

```diff
@@ -102,18 +102,26 @@
             <name>OpenUI5-Absa</name>
             <url>https://dl.bintray.com/absa/openui5</url>
         </repository>
+        <repository>
+            <id>confluent</id>
+            <url>https://packages.confluent.io/maven/</url>
+            <releases>
+                <updatePolicy>never</updatePolicy>
```

Comment:
I just took the definition from Abris as is. But this might speed up the build by not checking updates frequently. Once an artifact is downloaded we assume it is always recent. I guess...